frontend/src/models/study.ts

```diff
@@ -56,7 +56,7 @@ export const LaunchStudy = async (api: StudiesApi, study: {id: number}, options:
 
 export const isStudyLaunchable = (study: ParticipantStudy) => {
     return Boolean(
-        !study.completedAt &&
+        (!study.completedAt || (dayjs(study.completedAt).isSame(dayjs(study.optedOutAt), 'second'))) &&
```

Comment:
Had trouble comparing these timestamps in ruby, they differed in the millisecond range, so I went with a delta comparison method. That is more robust to "splitting the second", think I should use it here?

Reply:
Fixed in backend by making sure they're the same  timestamp (delta code had a bug, anyway)